### ISSUE TITLE
Not able to switch projects in darwin build

### ISSUE THREAD

**Issue by frankejj** [sapcc/kubernikus#476]
Hi,
We work in multiple projects and constantly need to switch environments.
We maintain openrc files with connection parameters.
Typical switch flow on mac os goes like this:
source project/openrc  #sets env vars
kubernikusctl auth init  #merges credentials to ~/.kube/config
....should now be authenticated....
if you do the same thing again to a different project it does not properly authenticate as expected.
this is only happening in recent versions of kubernikusctl compiled for darwin.
I rolled back to an old version and it works as expected.
Is there a different way we should be using kubernikusctl to authenticate or is this truly unexpected behavior?

**Comment by jknipper** [comment]
Hi frankejj,
I have a few questions to debug this:
- what combination of credentials you are using to authenticate against Openstack?
- is `kubernikusctl auth refresh` working?
- what do you mean by old version of kubernikusctl? I mean 3 month old, 1 year ...

**Comment by frankejj** [comment]
Hi,

- I use my user id and credentials that are provisioned on cc ldap.
- `kubernikusctl auth refresh` checks config and appears to work correctly but this does not seem relevant if I want to change my terminal context from one project to another.
- I am not sure exactly when the behavior started to occur. The version that I have that is working well for my use case is from Jan, 2018.  (an early release on github.com) -- It does not support `kubernikusctl version`

I only learned of the issue now because I don't regularly update this executable and I got a new macbook. Another colleague of mine also had the same issue the other day because he had a new version. It seems if you clear out .kube/config and start fresh it authenticates and works but then if you try to switch it doesn't behave as expected.
